corehq/apps/cloudcare/views.py

```diff
@@ -245,14 +246,16 @@ def form_context(request, domain, app_id, module_id, form_id):
 
 
 def get_cases_vary_on(request, domain):
+    request_params = request.GET if request.method == 'GET' else request.POST
```

Comment:
@nickpell I know you didn't want to add a helper method though it's not clear to me that having this line in a bunch of views is any better.

what if we had a function that was: `request_params = expect_GET(request)` that returns `request.GET`, but then soft asserts if it's a POST and returns `request.POST`. then we could put what we think it is everywhere and once there weren't any assertions we could fully switch over